test/prototype_transforms_kernel_infos.py

```diff
@@ -1263,7 +1263,9 @@ def _get_elastic_displacement(spatial_size):
 
 
 def sample_inputs_elastic_image_tensor():
-    for image_loader in make_image_loaders(sizes=["random"]):
+    for image_loader in make_image_loaders(
+        sizes=["random"], dtypes=[torch.uint8, torch.float16, torch.float32, torch.float64]
```

Comment:
```suggestion
        sizes=["random"], dtypes=[torch.uint8, torch.float32, torch.float64]
```
Shouldn't we remove f16 from here as well ?